File: 2016/test/day00_test.rb
# frozen_string_literal: true

require 'minitest/autorun'
require_relative 'util'

describe 'day00' do
  it 'solves part 01' do
    input = Util.read_input('day00-input.txt')

    assert_equal '0', input.first
  end
end

File: 2017/src/day00.test.ts
import { describe, test } from "node:test";
import { equal } from "node:assert";
import { readInput } from "./util.ts";

describe("day 00", () => {
  test("part 1", async () => {
    const input = await readInput("day00-input.txt");

    equal(input[0], "0");
  });
});
